biz.aQute.repository/test/aQute/bnd/repository/maven/provider/MavenBndRepoTest.java

```diff
@@ -81,6 +81,7 @@
 
 	@BeforeEach
 	public void setUp(TestInfo testInfo) throws Exception {
+		System.setProperty("org.slf4j.simpleLogger.defaultLogLevel", "DEBUG");
```

Comment:
Is this just something left over from your debugging? Seems like it should not be part of the PR.